website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -36,7 +36,7 @@ To create a new run task:
    - **Description** (optional): A human-readable description for the run task. This information can contain letters, numbers, spaces, and special characters.
    - **HMAC key** (optional): A secret key that may be required by the external service to verify request authenticity.
 
-5. Click **Create run task**. The run task is now available within the organization. You can now associate it with one or more workspaces.
+1. Click **Create run task**. The run task is now available within the organization. You can now associate it with one or more workspaces.
 
 ## Associating Run Tasks with a Workspace
```

Comment:
Maybe ... Perhaps we could surface our already listed integration partners more (from https://www.terraform.io/cloud-docs/integrations/run-tasks#run-tasks-technology-partners)

```
Check out our list of partner run task integrations for examples on how you can use run tasks
```

Reply:
@hcrhall Any thoughts here?

Reply:
I think that touching on the use case would be great, but I have concerns that this would set a precedent that would require us to document a use case for each stage which may be irritating if all you want to do is complete the setup. 

We can revisit this later once we have Run Tasks in the Registry and then direct folks there to discover integrations by stage, etc. Sending folks over to the partner list is not ideal because we don't have any partner-built integrations for Pre-Plan.

Reply:
Okay! This is a fair point and I accept that for now. I would love to revisit this once run tasks are in the registry! Thanks, @hcrhall!